packages/conf-radare2/conf-radare2.0.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+name: "conf-radare2"
+version: "0.1"
+maintainer: "Ivan Gotovchits <ivg@ieee.org>"
+authors: "BAP Team"
+homepage: "https://github.com/BinaryAnalysisPlatform/bap/"
+bug-reports: "https://github.com/BinaryAnalysisPlatform/bap/issues"
+dev-repo: "git://github.com/BinaryAnalysisPlatform/bap/"
+license: "MIT"
+build: [
+  ["ocaml" "unix.cma" "find-radare2.ml"]
+]
+depends: [
+   "ocaml"
+   "base-unix"
+   "conf-which" {build}
+]
+substs: [ "find-radare2.ml" ]
+flags: [ conf ]
+
+depexts: [ "radare2" ]
```

Comment:
we also depext on `conf-which`

Reply:
`conf-which` listed in `depends`, it's not a system dependency